LICENSE

```diff
@@ -1,339 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
We don't want a license in this repo?

Reply:
No, technically the license is overwritten by drupal core's license which is the most important thing and because WE are not specifically including a license we shouldn't include one in the repo especially because we want this to get updated any time we update drupal core